tensorboard/webapp/metrics/views/main_view/card_group_toolbar_component.scss

```diff
@@ -34,11 +34,16 @@ limitations under the License.
       cursor: pointer;
     }
   }
+
+  @include tb-dark-theme {
+    .group-toolbar {
+      color: map-get($tb-dark-foreground, text);
+    }
+  }
 }
```

Comment:
First, you are no longer allowed to use `map-get`. 
Second: can you just do `@include tb-theme-foreground-prop(color, text);` in L24?